src/v2/security/PERMISSION.ts

```diff
@@ -35,6 +35,7 @@ export enum PERMISSION {
   GROUPS_$GROUPID_TRANSFER = 'groups.$groupId.transfer',
   GROUPS_CAPTAIN_SWITCH = 'groups.captain.switch',
   GROUPS_$GROUPID_LIST_GET = 'groups.$groupId.list.get',
+  GROUP_LEAVE = 'groups.$groupId.leave',
```

Comment:
it’s better to name constant as `GROUPS_$GROUPID_LEAVE` so it will match the template